lib/presentation/widget/shared_space_details/shared_space_details_widget.dart

```diff
@@ -258,4 +267,71 @@ class _SharedSpaceDetailsWidgetState extends State<SharedSpaceDetailsWidget> {
       }
     );
   }
-}
+
+  Widget _activitiesTabWidget(List<AuditLogEntryUser> activitiesList) {
+    return activitiesList.isEmpty ?
+      SizedBox.shrink() :
+      ListView.builder(
+        key: Key('activities_list'),
+        padding: EdgeInsets.zero,
+        itemCount: activitiesList.length,
+        itemBuilder: (context, index) {
+          return _buildActivitiesListItem(context, activitiesList[index]);
+        },
+      );
+  }
+
+  Widget _buildActivitiesListItem(BuildContext context, AuditLogEntryUser auditLogEntryUser) {
+    return ListTile(
+        contentPadding: EdgeInsets.only(left: 24, top: 0),
+        leading: Column(
+            mainAxisAlignment: MainAxisAlignment.start,
+            children: [
+              SvgPicture.asset(auditLogEntryUser.getAuditLogIconPath(imagePath), width: 20, height: 20, fit: BoxFit.fill)
+            ]
+        ),
+        title: Transform(
+          transform: Matrix4.translationValues(-16, 0.0, 0.0),
+          child: Column(
+              crossAxisAlignment: CrossAxisAlignment.start,
+              children: [
+                Padding(
+                    padding: EdgeInsets.only(top: 8),
+                    child: Text(
+                      auditLogEntryUser.getResourceName(),
+                      maxLines: 1,
+                      style: TextStyle(fontSize: 14, color: AppColor.documentNameItemTextColor),
+                    )
+                ),
+                _buildActionDetailsText(auditLogEntryUser.getActionDetails(context, _model.store.state.account.user.userId.uuid)),
+                Padding(
+                    padding: EdgeInsets.only(top: 8),
+                    child: Text(
+                      auditLogEntryUser.getLogTimeAndByActor(context, _model.store.state.account.user.userId.uuid),
+                      style: TextStyle(fontSize: 13, color: AppColor.documentModifiedDateItemTextColor),
+                    )
+                )
+              ]
+          ),
+        )
+    );
+  }
+
+  Widget _buildActionDetailsText(Map<AuditLogActionDetail, dynamic> actionDetails) {
+    return Padding(
+        padding: EdgeInsets.only(top: 8),
+        child: RichText(
+            text: TextSpan(
+              style: TextStyle(fontSize: 13, color: AppColor.documentNameItemTextColor),
+              children: <TextSpan>[
+                TextSpan(
+                    text: actionDetails[AuditLogActionDetail.TITLE], style: TextStyle(fontWeight: FontWeight.bold, fontSize: 13, color: AppColor.documentNameItemTextColor)),
+                TextSpan(
+                    text: ' ' + actionDetails[AuditLogActionDetail.DETAIL],
```

Comment:
I think its more readable if you write like this `text: ' ${actionDetails[AuditLogActionDetail.DETAIL]}' ` . WDYT